build/common.props

```diff
@@ -57,9 +57,10 @@
     <InformationalVersion>$(Major).$(Minor).$(Revision).$(BuildNumber)$(PrereleaseLabel)</InformationalVersion>
     <PackageVersion>$(Major).$(Minor).$(Revision).$(BuildNumber)$(PrereleaseLabel)</PackageVersion>
   </PropertyGroup>
-  <PropertyGroup>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/Library/Frameworks/Mono.framework/Versions/Current')">/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/usr/local/lib/mono/xbuild-frameworks')">/usr/local/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-    <TargetFrameworkRootPath Condition="'$(TargetFrameworkRootPath)' == '' AND Exists('/usr/lib/mono/xbuild-frameworks')">/usr/lib/mono/xbuild-frameworks</TargetFrameworkRootPath>
-  </PropertyGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('net4')) ">
```

Comment:
You don't need condition here. It handled into ref asm packages.

Reply:
Force pushed and fixed!